### ISSUE TITLE
Orientation sensor doen't work properly

### ISSUE THREAD

**Issue by DerSaxxe** [piccolo-dev/android_device_bq_piccolo#25]
In version from 21.12.2015 the orientation doesn't work properly. After manual reboot it works a while.
Thank you

PS: Bootloader is the new one.

**Comment by piwats** [comment]
Can confirm. After some time it gets stuck.

**Comment by piwats** [comment]
Also the magnetic sensor seems to be affected as well. Compass lags or stays stuck.

**Comment by Kra1o5** [comment]
Can you guys provide a logcat/dmesg?

Thanks

**Comment by piwats** [comment]
will do ASAP

**Comment by DerSaxxe** [comment]
[dmesg.zip](https://github.com/piccolo-dev/android_device_bq_piccolo/files/70966/dmesg.zip)
Here is it... 

Kind regards
Ronny Steiner

Am 23. Dezember 2015 13:34:35 MEZ, schrieb Alejandro Lopez notifications@github.com:

**Comment by Kra1o5** [comment]
@SIRSteiner,

Thanks but not help me much.

I think this problem is selinux related.

@eloimuns can you set selinux to permissive to test it?

**Comment by DerSaxxe** [comment]
Can I help with other commands on terminal as root? 

Kind regards
Ronny Steiner

**Comment by Kra1o5** [comment]
@SIRSteiner , yes.

But it can be more easy using terminal on PC with:

adb pull sepolicy; adb shell su -c dmesg | audit2allow -p sepolicy

Thanks

**Comment by DerSaxxe** [comment]
Here is the first...
Is the second command correct? Pipe in adb shell? I get bash: audit2allow: command not found

Sorry I have to send sepolicy per mail. Upload not possible...
Just a moment...

**Comment by Kra1o5** [comment]
To use audit2allow you need to have it installed on linux pc.

sepolicy file is needed for compare sepolicy & dmesg.

**Comment by DerSaxxe** [comment]
I use manjaro. No package found for that. But should I send you the sepolicy and the output of
adb shell su -c dmesg
?

I did it. Look at your mail inbox please

**Comment by Kra1o5** [comment]
You need install: policycoreutils

**Comment by piwats** [comment]
From a fresh reinstalled 3.2.2 and then converted to a clean cm-20151221:

audit2allow gives this out:

```
libsepol.policydb_read: policydb magic number 0x5b3e363c does not match expected magic number 0xf97cff8c or 0xf97cff8d
```

**Comment by DerSaxxe** [comment]
Thank you piwats, because I am not able to install policycoreutils. It is only available in AUR and there are some errors while compiling dependencies, sorry

**Comment by piwats** [comment]
I think my kernel is too new to do a audit2allow. Mine is 4.3.3 vs 3.10.49 on the phone.

**Comment by DerSaxxe** [comment]
I sent Kra1o5 both sepolicy and the dmesg output. Maybe he can do
cat dmesg.txt | audit2allow -p sepolicy
with my files?

**Comment by piwats** [comment]
I made a video showing the compass being stuck/lagging, if needed I can upload it.

**Comment by Kra1o5** [comment]
Fixed in cm-12.1-20151227-UNOFFICIAL-piccolo.

Greetings

**Comment by stucki** [comment]
This issue was moved to bq-dev/android_device_bq_piccolo#25
